.ci/resolve-kata-dependencies.sh

```diff
@@ -62,13 +62,14 @@ clone_repos() {
 		echo "Cloning ${repo}"
 		go get -d "${repo}" || true
 		repo_dir="${GOPATH}/src/${repo}"
+		pushd "${repo_dir}"
 
-		# The tests repository is cloned and checkout to the PR branch directly in
-		# the CI configuration (e.g. jenkins file or zuul config), because we want
-		# to have latest changes of this repository, since the job starts. So we
-		# need to verify if we are already in the PR branch, before trying to
-		# fetch the same branch.
-		if [ ${repo} == ${tests_repo} ]
+		# When we have a change from the tests repo, the tests repository is cloned
+		# and checkout to the PR branch directly in the CI configuration (e.g. jenkins
+		# config file or zuul config), because we want to have latest changes
+		# of this repository since the job starts. So we need to verify if we
+		# are already in the PR branch, before trying to fetch the same branch.
+		if [ ${repo} == ${tests_repo} ] && [ "${repo}" == "${kata_repo}" ]
```

Comment:
Nit: might be worth adding a comment on the first use of the `kata_repo` variable (line 18?) stating this is set by Jenkins.

Reply:
ack - let's discuss/expand this a touch. The line confuses me a little. If I'm reading that right:
```
if [ ${repo} == ${tests_repo} ] && [ "${repo}" == "${kata_repo}" ]
```

then kata_repo must be the same as tests_repo for this to pass? I'm a bit confused about the difference between `repo` and `kata_repo`.